tutorial/source/index.rst

```diff
@@ -3,92 +3,202 @@
+
+List of Tutorials
+==================
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Introduction:
+   :maxdepth: 1
+   :caption: Introductory Tutorials
+   :name: introductory-tutorials
 
    intro_part_i
    intro_part_ii
    svi_part_i
    svi_part_ii
    svi_part_iii
-   tensor_shapes
-   mle_map
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Advanced:
+   :maxdepth: 1
+   :caption: Hands On with Pyro and PyTorch
+   :name: hands-on-with-pyro-and-pytorch
 
-   enumeration
-   custom_objectives
-   jit
-   minipyro
-   effect_handlers
+   bayesian_regression
+   bayesian_regression_ii
+   tensor_shapes
    modules
-   normalizing_flows_i
-   contrib_funsor_intro_i
-   contrib_funsor_intro_ii
+   jit
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Examples:
+   :maxdepth: 1
+   :caption: Deep Generative Models
+   :name: deep-generative-models
 
    vae
-   bayesian_regression
-   bayesian_regression_ii
-   dmm
-   air
    ss-vae
-   stable
    cvae
+   normalizing_flows_i
+   dmm
+   air
+   cevae
+   sparse_gamma
+   svi_horovod
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Contributed:
+   :maxdepth: 1
+   :caption: Discrete Latent Variables
+   :name: discrete-latent-variables
 
+   enumeration
    gmm
-   gp
-   gplvm
-   bo
+   dirichlet_process_mixture
+   toy_mixture_model_discrete_enumeration
+   hmm
+   capture_recapture
+   einsum
+   lda
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Customizing Inference
+   :name: customizing-inference
+
+   mle_map
    easyguide
+   custom_objectives
+   boosting_bbvi
+   neutra
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Application: Time Series
+   :name: time-series
+
    forecasting_i
    forecasting_ii
    forecasting_iii
-   tracking_1d
-   csis
-   RSA-implicature
-   RSA-hyperbole
-   ekf
-   working_memory
-   elections
-   dirichlet_process_mixture
-   boosting_bbvi
-   epi_intro
+   forecasting_dlm
+   stable
+   forecast_simple
+   timeseries
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Code Examples:
+   :maxdepth: 1
+   :caption: Application: Gaussian Processes
+   :name: gaussian-processes
 
-   capture_recapture
-   cevae
-   hmm
-   lda
-   mcmc
-   neutra
-   sparse_gamma
+   gp
+   gplvm
+   bo
    dkl
-   einsum
-   forecast_simple
-   timeseries
-   smcfilter
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Application: Epidemiology
+   :name: epidemiology
+
+   epi_intro
    epi_sir
    epi_regional
    sir_hmc
-   toy_mixture_model_discrete_enumeration
-   svi_horovod
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Application: Optimal Experiment Design
+   :name: optimal-experiment-design
+
+   working_memory
+   elections
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Application: Object Tracking
+   :name: object-tracking
+
+   tracking_1d
+   ekf
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Other Examples:
+
+   RSA-implicature
+   RSA-hyperbole
+   mcmc
+   csis
+   smcfilter
```

Comment:
Hmm should we group inference algorithms together? It looks like we currently focus on 
SVI and "customizing inference" but don't mention MCMC or SMC except as miscellaneous. Should we treat SMC as "an example of custom inference implemented using poutines" or should we treat it as a standalone inference algorithm? Maybe it should be in the custom inference section?

Reply:
I'm agnostic to this in the abstract but the problem is that these other examples are just not that helpful compared to our extensive material on SVI.  I'll rename this section "other inference algorithms" rather than put them in "customizing inference."